backend/clickhouse/cursor.go

```diff
@@ -47,18 +47,18 @@ func getConnection[T interface{}](edges []*Edge[T], pagination Pagination) *Conn
 		}
 	} else if pagination.After != nil && len(*pagination.After) > 1 {
 		hasPreviousPage = true // implicitly true because the passed in cursor should match
```

Comment:
Technically, these changes to the conditional logic here shouldn't be necessary with the `LIMIT` on the outer query, but I didn't see how they hurt. @Vadman97 let me know if you think we should keep them around or revert.

Reply:
sounds good to keep them!